spec/models/keyword_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Keyword, type: :model do
+  describe 'Validations' do
+    it { is_expected.to validate_presence_of(:name) }
+
+    context 'given a blank name' do
+      it 'raises a RecordInvalid error' do
+        expect { Fabricate(:keyword, name: " \n") }.to raise_error(ActiveRecord::RecordInvalid)
+      end
+    end
+
+    context 'given a too long name' do
```

Comment:
Can we do this via 
```
   it { is_expected.to validate_length_of(:name).is_greater_than(5).is_at_most(19)
```

https://github.com/thoughtbot/shoulda-matchers/blob/master/lib/shoulda/matchers/active_model/validate_length_of_matcher.rb

Reply:
YAD! (Yet Another Discovery)
Thanks! :) 

Fixed~